docs/src/pages/docs/crocks/Async.md

```diff
@@ -1257,6 +1257,69 @@ timeout(slow)
 
 ## Transformation Functions
 
+#### asyncToPromise
```

Comment:
Need to add `asynctoproimse` to the functions collection at the top of this file

Reply:
done